scripts/deploy-federation.sh

```diff
@@ -122,14 +122,14 @@ if [[ "${NAMESPACED}" ]]; then
   kubectl -n "${NS}" create rolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
 else
   # TODO(marun) Make this more restrictive.
-  kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  kubectl create clusterrolebinding federation-admin-${NS} --clusterrole=cluster-admin --serviceaccount="${NS}:default"
 fi
 
 if [[ ! "${USE_LATEST}" ]]; then
   if [[ "${NAMESPACED}" ]]; then
     INSTALL_YAML="${INSTALL_YAML}" IMAGE_NAME="${IMAGE_NAME}" scripts/generate-namespaced-install-yaml.sh
   else
-    INSTALL_YAML="${INSTALL_YAML}" IMAGE_NAME="${IMAGE_NAME}" scripts/generate-install-yaml.sh
+    INSTALL_YAML="${INSTALL_YAML}" IMAGE_NAME="${IMAGE_NAME}" NS="${NS}" scripts/generate-install-yaml.sh
```

Comment:
Just like in any programming langauge, it's fine to use a short name internally but the API exposed to the user should be appropriately descriptive.  That suggests `FEDERATION_NAMESPACE` as the environment variable the generation script accepts, not `NS`.